protocols/bgp/server/fsm.go

```diff
@@ -242,11 +243,38 @@ func (fsm *FSM) cease() {
 	fsm.eventCh <- Cease
 }
 
+func (fsm *FSM) sockSettings(c net.Conn) error {
+	ttl := fsm.peer.ttl
+	setNoRoute := false
+	if ttl == 0 {
+		if fsm.peer.isEBGP() {
```

Comment:
Requires user of eBGP multihop to set an TTL explicitly

Reply:
That's intended.